### PR TITLE
:arrow_up::one::three: auth_quick_master

### PR DESCRIPTION
**Pull request by trojikman**

### CODE REVIEW

**Review by yelizariev — APPROVED** [review]
No changes in code

File: auth_quick_master/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+# Copyright 2019 Denis Mudarisov <https://it-projects.info/team/trojikman>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     "name": """Quick Auth (Master)""",
     "summary": """Authentication provider for "Quick Authentication" module""",
     "category": "Extra Tools",
     # "live_test_url": "http://apps.it-projects.info/shop/product/DEMO-URL?version=12.0",
-    "images": ['images/quick_auth_mater.jpg'],
-    "version": "12.0.1.1.0",
+    "images": [],
```
Comment (ilmir-k) [inline auth_quick_master/__manifest__.py]: @trojikman we have the main image for this module, it should link to ``images/quick_auth_master.jpg``

Comment (trojikman) [reply auth_quick_master/__manifest__.py]: Done

Comment (ilmir-k) [reply auth_quick_master/__manifest__.py]: @trojikman fix typo, it should be 
quick_auth_ma**s**ter.jpg

**Comment by ilmir-k** [comment]
I approve to merge it now

**Pull request merged**